app/views/swiss/home.scala

```diff
@@ -59,7 +59,8 @@ object home {
               span(cls := "rounds")(
                 s.isStarted option frag(s.round.value, " / "),
                 s.settings.nbRounds,
-                " rounds Swiss"
+                " ",
+                trans.swiss.roundsSwiss()
```

Comment:
This probably should use a placeholder. Probably even a plural since some languages might differentiate between <=4 and >4 rounds or similar numbers.

Reply:
I am not really convinced plurals are relevant for the translation here; we won't ever have  1 round swiss.

Reply:
> since some languages might differentiate between <=4 and >4 rounds or similar numbers.

🤔

Reply:
Even more than I could imagine 👀